redir.php

```diff
@@ -1,9 +1,36 @@
 <?php
+// URL to list of CSV Shortlinks
+$URLToCSV="https://raw.githubusercontent.com/hyphacoop/shortlinks/master/shortlinks.csv";
 
+// Enable caching of file by defining the number of minutes the file is considered stale
+// 0 = disables caching
+$cacheInterval=15;
```

Comment:
I wonder if there can be a way to force refresh of cache immediately

Reply:
A force refresh warms me to this cache idea

Reply:
```suggestion
$cacheInterval=15;

$forceRefresh=0;
if (isset($_REQUEST['force'])) $forceRefresh = 1;
```